lib/commands/doctor/checks/check-memory.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const os = require('os');
+const SystemError = require('../../../errors').SystemError;
+
+const MB_IN_BYTES = 1048576;
+const MIN_MEMORY = 150;
+
+function checkMemory(ctx, task) {
+    if (!ctx.argv['check-mem']) {
+        return task.skip();
```

Comment:
I wonder if it's worth moving this to an `enable` function? That way the check wouldn't show up in the list if it's been intentionally ignored 🤔

Reply:
Done!